manifests/cli.pp

```diff
@@ -36,7 +32,7 @@
     creates => $jar,
   }
   ~> exec { 'jenkins-cli' :
-    command     => "${extract_jar} && ${move_jar} && ${remove_dir}",
+    command     => "sleep 30 && ${extract_jar}",
```

Comment:
Why `sleep 30`?

Reply:
As it is downloading the "jenkins-cli" right after the Jenkins service is activated, we need to wait for a couple of seconds so the path will be accessible.